docs/trace/customizing-the-sdk/README.md

```diff
@@ -620,21 +620,16 @@ components.
 Options classes can always be configured through code but users typically want to
 control key settings through configuration.
 
-The following example shows how to configure `JaegerExporterOptions` by binding
+The following example shows how to configure `ZipkinExporterOptions` by binding
```

Comment:
Not a blocker, I think we want to encourage OTLP by default?

Reply:
Updated to OTLP exporter.